project-backend/Models/JobController/GetJobs/GetJobsResponseObject.cs

```diff
@@ -5,14 +5,33 @@ namespace project_backend.Models.JobController.GetJobs
 {
     public class GetJobsResponseObject
     {
+        public class FetchedUser
+        {
+            public string FirstName { get; set; }
+            public string LastName { get; set; }
+            public string ProfileImage { get; set; }
+        }
+
+        public class FetchedWorker
+        {
+            public FetchedUser User { get; set; }
+        }
+
+        public class FetchedBid
+        {
+            public FetchedWorker Worker { get; set; }
```

Comment:
I would lean towards also using FetchedUser here. Would get rid of one level of unnecessary nesting.

Reply:
I can change it of course, but that would require some changes on the frontend.

Reply:
Ok, then it's fine I guess. I didn't necessarily want the same class to be used, just to get rid of that nesting. If the frontend already implemented this object like this, it's alright.